src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
 // write code here
+
+const spider = document.querySelector('.spider');
+
+spider.style.top = '50%';
+spider.style.left = '50%';
+spider.style.transform = 'translate(-50%, -50%)';
```

Comment:
that's nice to set it by pure CSS, it works) But the idea of the task is to calculate the correct position using JS - you should also query `.wall` element to do it